css/selectors/attribute.json

```diff
@@ -103,16 +103,31 @@
             "description": "Case-sensitive modifier (<code>s</code>)",
             "support": {
               "chrome": {
-                "version_added": false,
-                "notes": "See <a href='https://crbug.com/1041095'>bug 1041095</a>."
+                "version_added": "90",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "CSSCaseSensitiveSelector"
```

Comment:
I hadn't seen a flag named like this before and didn't find it in the UI. Turns out, this is the feature name in [`runtime_enabled_features.json5`, where it's a "test"](https://github.com/chromium/chromium/blob/master/third_party/blink/renderer/platform/runtime_enabled_features.json5#L479). As far as I can tell, such features can't be enabled in regular Chrome releases. I don't think BCD has a way to represent this yet (perhaps someday with https://github.com/mdn/browser-compat-data/pull/7396). I'd welcome a return of this PR, when/if Chrome supports this in a way that web developers can actually try out. Thanks, @ExE-Boss!